src/crimson/osd/pg.cc

```diff
@@ -865,4 +865,50 @@ void PG::handle_rep_op_reply(crimson::net::Connection* conn,
   backend->got_rep_op_reply(m);
 }
 
+void PG::request_replica_scan(
```

Comment:
This is an artifact from the past. Backfill has been forked form the recovery branch when it wasn't using `PGRecovery` yet. Need to move it out.